spec/controllers/settings_controller_spec.rb

```diff
@@ -27,7 +27,7 @@
       end
     end
   end
-
```

Comment:
There needs to be something added to the `#twitter` action for adding the UID to a currently signed in user.  Also, tests.